src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -791,34 +791,34 @@ static function (ChildDefinition $definition, AsPeriodicTask|AsCronTask $attribu
         }
 
         $container->registerForAutoconfiguration(CompilerPassInterface::class)
-            ->addTag('container.excluded', ['source' => 'because it\'s a compiler pass'])->setAbstract(true);
+            ->addTag('container.excluded', ['source' => 'because it\'s a compiler pass']);
         $container->registerForAutoconfiguration(Constraint::class)
-            ->addTag('container.excluded', ['source' => 'because it\'s a validation constraint'])->setAbstract(true);
+            ->addTag('container.excluded', ['source' => 'because it\'s a validation constraint']);
         $container->registerForAutoconfiguration(TestCase::class)
-            ->addTag('container.excluded', ['source' => 'because it\'s a test case'])->setAbstract(true);
+            ->addTag('container.excluded', ['source' => 'because it\'s a test case']);
         $container->registerForAutoconfiguration(\UnitEnum::class)
-            ->addTag('container.excluded', ['source' => 'because it\'s an enum'])->setAbstract(true);
+            ->addTag('container.excluded', ['source' => 'because it\'s an enum']);
         $container->registerAttributeForAutoconfiguration(AsMessage::class, static function (ChildDefinition $definition) {
-            $definition->addTag('container.excluded', ['source' => 'because it\'s a messenger message'])->setAbstract(true);
+            $definition->addTag('container.excluded', ['source' => 'because it\'s a messenger message']);
         });
         $container->registerAttributeForAutoconfiguration(\Attribute::class, static function (ChildDefinition $definition) {
-            $definition->addTag('container.excluded', ['source' => 'because it\'s an attribute'])->setAbstract(true);
+            $definition->addTag('container.excluded', ['source' => 'because it\'s an attribute']);
```

Comment:
```suggestion
            $definition->addTag('container.excluded', ['source' => 'because it\'s a PHP attribute']);
```

"attribute" could be several stuff nowadays 😄